modules/config_games/server_configs/valheim_linux64.xml

```diff
@@ -0,0 +1,40 @@
+<game_config>
+ <game_key>valheim_linux</game_key>
+   <protocol>lgsl</protocol>
+  <lgsl_query_name>valheim</lgsl_query_name>
+ <installer>steamcmd</installer>
+ <game_name>Valheim</game_name>
+ <server_exec_name>valheim_server.x86_64</server_exec_name>
+ <cli_template>-name %HOSTNAME% %PORT% %HOME_PATH% %WORLD% %PASSWORD% -public= 1 </cli_template>
```

Comment:
I see blank spaces at the end of the line, is it correct? doesn't seem like it is to me without searching documentation